mobile/.bazelrc

```diff
@@ -67,6 +68,10 @@ build:dbg-android --config=android
 build:dbg-ios --config=dbg-common
 build:dbg-ios --config=ios
 
+# Default flags for Android tests
+# TODO(jpsim): Explicitly register test extensions for Android tests
+build:android-test --define=static_extension_registration=enabled
```

Comment:
Most of our configs use the naming convention where the name of the platform is the suffix: release-android, release-common, dbg-android etc. Maybe we should call this new config `test-android` or `ci-test-android`? 

 While on this, could we just use add `static_extension_registration=enabled` to `dbg-android` and use `dbg-android` in tests? Why did we decided to add a separate config in here?

Reply:
Do we have an issue that tracks an effort of disabling `static_extension_registration` in Android tests?